helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -805,6 +805,10 @@ service:
         {{- end }}
         - resourcedetection
         - resource
+        {{/*
+        `deployment.environment` atributes is not being set on metrics sent to Splunk Observability because it's already synced as `sf_environment` property.
```

Comment:
```suggestion
        The attribute `deployment.environment` is not being set on metrics sent to Splunk Observability because it's already synced as the `sf_environment` property.
```